m2cgen/ast.py

```diff
@@ -141,6 +141,17 @@ class CompOpType(Enum):
     EQ = '=='
     NOT_EQ = '!='
 
+    @staticmethod
+    def from_str_op(op):
+        return {
+            ">": CompOpType.GT,
```

Comment:
I think this can be slightly improved:
```
class CompOpType(Enum):
    GT = '>'
    GTE = '>='
    LT = '<'
    LTE = '<='
    EQ = '=='
    NOT_EQ = '!='

    @staticmethod
    def from_str_op(op):
        return COMP_OP_TYPE_MAPPING[op]


COMP_OP_TYPE_MAPPING = {e.value: e.name for e in CompOpType}
```

Reply:
Thanks!